### ISSUE TITLE
Hightlighted source code contrast is illegible in night mode

### ISSUE THREAD

**Issue by danielzgtg** [slapperwan/gh4a#1274]
It is almost impossible to read highlighted source code in night mode.

Example link: https://github.com/linux-surface/iptsd/blob/master/etc/iptsd.conf#L62-L78 or more permanently https://github.com/linux-surface/iptsd/blob/3ec9be098bd42d8fa1f7f49c13689b174edb30cb/etc/iptsd.conf#L62-L78

# Expected Behavior

Text contrast should follow accessibility guidelines. It should be white when the highlight is dark yellow.

# Actual Behavior

Grey text on dark yellow highlight is unreadable:

![Screenshot_20230226-060329_OctoDroid](https://user-images.githubusercontent.com/25646384/221406732-63c6443f-29de-4f3b-b387-94e51cc08958.png)
